themes/digital.gov/src/scss/new/_uswds-overrides.scss

```diff
@@ -47,3 +47,12 @@
 .dg-guide-summary {
   margin-bottom: units(2);
 }
+
+// Archived alerts
+// Seen on: `2011/06/21/making-mobile-gov-project`
+.dg-alert--out-of-date {
+  .usa-alert__body::before {
+    background-image: url("../img/svg-icons/material-icon-inventory_2.svg"); // Fallback for mask-image
+    mask-image: url("../img/svg-icons/material-icon-inventory_2.svg");
```

Comment:
I'm curious why we would prefer to use `mask-image` as the main rule for setting the icon? I haven't seen that before.

Reply:
This is the technique the component uses. It allows us to set the icon as a background image, but also leverage the benefits of an SVG and change the color without creating a duplicate.